packages/venia-concept/src/signoutExtension.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * This is a mock extension. Assume this file will be provided by
+ * an extension NPM package. For now for the simplicity of the POC
+ * I have added this file as part of venia-concept.
+ */
+
+export default tapableHooks => {
+    tapableHooks.syncOnSignOut.tap('syncOnSignOut', () => {
```

Comment:
So this here is external/vendor code that a client would import/install. We will need to publish the supported callback names as well as their [supported methods](https://github.com/webpack/tapable#hookhookmap-interface).

Reply:
Yes. It is assumed that the extension provider will have a doc or a page to let the user know about these hooks.